sample/TKNotificationPollSamples.m

```diff
@@ -0,0 +1,77 @@
+//
+//  TKNotificationPollSamples.m
+//  TokenSdkTests
+//
+//  Created by Larry Hosken on 11/8/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "TKTestBase.h"
+
+#import "TokenSdk.h"
+
+@interface TKNotificationPollSamples : TKTestBase
+
+@end
+
+@implementation TKNotificationPollSamples
+
+-(void)testNotificationPolling {
+    TokenIOSync *tokenIOSync = [[self sdkBuilder] buildSync];
+    Alias *payerAlias = [self generateAlias];
+    TKMemberSync *payerSync = [tokenIOSync createMember:payerAlias];
+    [payerSync linkAccounts:[self createBankAuthorization:payerSync]];
+    TKMember *payer = payerSync.async;
+    
+    __block Subscriber *subscriber = nil;
+    
+    [payer subscribeToNotifications:@"iron"
+                handlerInstructions:(NSMutableDictionary<NSString *, NSString *> *) [NSMutableDictionary dictionary]
+                          onSuccess:^(Subscriber *s) {
+                              subscriber = s;
+                          } onError:^(NSError *e) {
+                              // Something went wrong.
+                              @throw [NSException exceptionWithName:@"SubscribeException"
+                                                             reason:[e localizedFailureReason]
+                                                           userInfo:[e userInfo]];
+                          }];
+    
+    [self runUntilTrue:^ {
+        return (subscriber != nil);
+    }];
+    
+    // generate a notification so that our polling finds something...
+    TokenPayload *payload = [TokenPayload message];
+    payload.from.alias = payerAlias;
+    payload.to.id_p = [self createAccount:tokenIOSync].member.id;
+    payload.transfer.lifetimeAmount = @"100";
+    payload.transfer.currency = @"EUR";
+    [tokenIOSync notifyPaymentRequest:payload];
```

Comment:
if this section of code will be shown in the doc, maybe notification "PAYER_TRANSFER_PROCESSED" is better.
you can create this create and redeem a transfer token. since the notify request payment method is used by our apps only afaik.

Reply:
good idea. oh... now that you mention it , a developer wanted PAYEE_TRANSFER_PROCESSED for their flow. (they are payee, but don't create/endorse/redeem token, and never got an id for token/transfer/transaction/anything.)

(hmm, now I want to change the java and js examples to use this also...)